en/docs/chapter_searching/searching_algorithm_revisited.md

```diff
@@ -1,48 +1,48 @@
 # Search algorithms revisited
 
-<u>Searching algorithms (searching algorithm)</u> are used to search for one or several elements that meet specific criteria in data structures such as arrays, linked lists, trees, or graphs.
+<u>Searching algorithms (search algorithms)</u> are used to retrieve one or more elements that meet specific criteria within data structures such as arrays, linked lists, trees, or graphs.
 
-Searching algorithms can be divided into the following two categories based on their implementation approaches.
+Searching algorithms can be divided into the following two categories based on their approach.
 
```

Comment:
'implementation approaches' is redundant (it says the same thing twice). We could also say 'based on their implementation.'